tools/topology/topology1/CMakeLists.txt

```diff
@@ -173,6 +173,7 @@ set(TPLGS
 	"sof-imx8mp-wm8960\;sof-imx8mp-drc-wm8960\;-DPPROC=drc"
 
 	"sof-mt8195-mt6359-rt1019-rt5682\;sof-mt8195-mt6359-rt1019-rt5682"
+	"sof-acp-renoir\;sof-acp"
```

Comment:
add newline above to clearly split Mediatek and AMD topologies please

Reply:
Addressed the review comment.